newIDE/app/src/EventsSheet/index.js

```diff
@@ -397,8 +400,11 @@ export default class EventsSheet extends React.Component<Props, State> {
       );
     }
 
+    console.log(instructionContext, this.state);
     this.setState({
+      inlineEditingAnchorEl: instructionContext.domEvent,
       editedInstruction: {
+        quick: instructionContext.quick,
```

Comment:
"quick" is a bit generic, I would go for something more specific like: `isInlineInstructionEditor`. Easier to search in the codebase too and you immediately understand it's related to "an editor of instruction that should be shown inline or not". Finally the "is" hints as this being a boolean.

Reply:
Ah actually you're storing a DOM event there! Found that by reading the code (good example about avoiding short names ;)).

In this case, I would rather avoid storing DOM events as much as possible - it's "leaking" too much information that is potentially useless.
Instead of `quick`, I would pass back from the "Add Action" button the fact that we're requiring an inline instruction editor (`isInlineInstructionEditor`, a boolean) and the position on screen: `e.clientX, e.clientY`, which can be stored in `inlineInstructionEditorX` and `inlineInstructionEditorY`.

This is what we do for menus. Note that X/Y means that this won't stay at the proper position if we scroll... but I think it's not a big deal and we should maybe disable scroll when a menu/popover is opened. It might be already the case actually.

Reply:
I think it already is the target, but you're right the name is awful.
quickAddMenuTarget sound better?

Reply:
using onInstructionsListContextMenu to trigger the event would remove the need to add this target I think

Reply:
actually I kept the target in order to keep consistency